docs/entities/post-conditions/post-condition-0-stx.schema.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "PostConditionStx",
+  "type": "object",
+  "allOf": [
+    {
+      "type": "object",
+      "required": ["principal"],
+      "properties": {
+        "principal": {
+          "$ref": "./post-condition-principal.schema.json"
+        }
+      }
+    },
+    {
+      "type": "object",
+      "required": ["condition_code", "amount", "asset_info_id"],
```

Comment:
Mismatch with `condition_code` here. A tx has a single condition code, then an array of post conditions (as opposed to each post condition having a condition code)

Reply:
nvm! I mixed up condition code and condition mode!